pages/people/[id]/index.tsx

```diff
@@ -1,21 +1,93 @@
-import { useRouter } from 'next/router';
+import { getResident } from 'lib/residents';
+import Layout from 'components/NewPersonView/Layout';
+import { GetServerSideProps } from 'next';
+import { Resident, User } from 'types';
+import { useCasesByResident } from 'utils/api/cases';
+import { Case } from 'types';
+import PersonTimeline from 'components/NewPersonView/PersonTimeline';
+import Spinner from 'components/Spinner/Spinner';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import { useUnfinishedSubmissions } from 'utils/api/submissions';
+import { canManageCases } from 'lib/permissions';
+import { useAuth } from 'components/UserContext/UserContext';
+import { isAuthorised } from 'utils/auth';
 
-import Seo from 'components/Layout/Seo/Seo';
-import { PersonDetailsPage } from 'components/pages/people/PersonDetailsPage';
+interface Props {
+  person: Resident;
+}
 
-const PersonPage = (): React.ReactElement => {
-  const { query } = useRouter();
+const PersonPage = ({ person }: Props): React.ReactElement => {
+  const {
+    data: casesData,
+    size,
+    setSize,
+    error: casesError,
+  } = useCasesByResident(person.id);
+  const { data: submissionsData } = useUnfinishedSubmissions(person.id);
 
-  const personId = Number(query.id as string);
+  // flatten pagination
+  const events = casesData?.reduce(
+    (acc, page) => acc.concat(page.cases as Case[]),
+    [] as Case[]
+  );
+
+  const onLastPage = !casesData?.[casesData.length - 1].nextCursor;
 
   return (
-    <>
-      <Seo title={`Person Details - #${query.id}`} />
-      <PersonDetailsPage personId={personId} />
-    </>
+    <Layout person={person}>
+      {events ? (
+        events.length > 0 ? (
+          <PersonTimeline
+            unfinishedSubmissions={submissionsData?.submissions}
+            events={events}
+            size={size}
+            setSize={setSize}
+            onLastPage={onLastPage}
+          />
+        ) : (
+          <p>No events to show</p>
+        )
+      ) : casesError ? (
+        <ErrorMessage label={casesError.message} />
+      ) : (
+        <Spinner />
+      )}
+    </Layout>
```

Comment:
What do you think about feature flagging this? So if it's on, use this improved person view and if it's not, use the old `PersonDetailsPage`. This is quite a big visual change so I wonder if we'll need approval before it goes out but tbh I think it's always good to put a feature flag just in case.